index.php

```diff
@@ -9,28 +9,34 @@
 $locale=json_decode(file_get_contents("locales/".$user_locale.".json"), true);
 
 // Parse the apps list
-$apps_list_json=file_get_contents("https://app.yunohost.org/apps.json");
+$apps_list_json=file_get_contents("http://app.yunohost.org/apps.json");
```

Comment:
Why http?

Reply:
Ah yes sorry ... that's because we have this deployed in a LXC behind a reverse proxy and this domain is hosted on the same LXC ... and basically we end up with an invalid cert inside the LXC itself and have to do the request in http ...

I think the "real" fix for this would be to re-do our reverse-proxy setup to use proper SNI-based reverse proxy so that the LXC can handle Let's Encrypt like they should

Reply:
(If you look into the code currently deployed, there are uncommited changed to make it happen in http already)

Reply:
whoukay